frontend_tests/casper_tests/00-realm-creation.js

```diff
@@ -0,0 +1,91 @@
+var common = require('../casper_lib/common.js').common;
+
+var email = 'alice@example.com';
+var domain = 'example.com';
```

Comment:
let's use e.g. `@test.example.com` here -- we're planning to change the default Shakespeare users to be in the @example.com organization (https://github.com/zulip/zulip/issues/1118)), and I'd prefer not to have this break when we do that :)